Detections/DSMAzureBlobStorageLogs/AccessToSensitiveDataByTerminatedEmployees.yaml

```diff
@@ -0,0 +1,88 @@
+id: 5feef89a-d632-4294-a05c-988a1c64da23
+name: Access to sensitive data by Terminated Employees
+description: |
+  This query detects access to data that has classification insights (link below) and/or sensitivity label insights (link below), by users that are marked as terminated employees.
+  Classification insights: https://docs.microsoft.com/en-us/azure/purview/classification-insights
+  Sensitivty label insigths: https://docs.microsoft.com/en-us/azure/purview/sensitivity-insights
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
```

Comment:
ThreatIntelligence  connectorId is used here, but it doesn't seem like we are using the Threat Intelligence table in the query.  Can you please double check.

Reply:
I guess that answers my other comment :)